src/app/components/GitHubLinkExtension.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Image from 'terra-image';
+
+const propTypes = {
+  /**
+   * The github url to navigate to when the link is clicked.
+   */
+  href: PropTypes.string,
+};
+
+const defaultProps = {
+  href: undefined,
+};
+
+const GithubLinkExtension = (props) => {
+  const { href } = props;
+  return (<a href={href}>
+    <Image
+      variant="rounded"
+      src={'https://github.com/cerner/terra-dev-site/raw/master/src/app/components/github-logo.png'}
```

Comment:
Anything special copyright wise associated with this image?

Reply:
We should be good. 
>Use the Octocat or GitHub logo to link to GitHub

https://github.com/logos